constructorio-client/src/main/java/io/constructor/client/models/BrowseResponseInner.java

```diff
@@ -0,0 +1,51 @@
+package io.constructor.client.models;
+
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Constructor.io Browse Response Inner ... uses Gson/Reflection to load data in
+ */
+public class BrowseResponseInner {
+
+    @SerializedName("facets")
+    List<FilterFacet> facets;
+
+    @SerializedName("groups")
+    private List<FilterGroup> groups;
+
+    @SerializedName("results")
+    private List<Result> results;
+
+    @SerializedName("total_num_results")
+    private Integer totalNumberOfResults;
+
```

Comment:
Do we also want to add in `sort_options` as well?

If so, we should add it in for Search as well. 👍

Reply:
Let's do that in the story for variations?